### ISSUE TITLE
Bus error: 10

### ISSUE THREAD

**Issue by Paleve** [eragonruan/text-detection-ctpn#36]
When I run the demo.py, it appears this error:
Loading network VGGnet_test...  Restoring from checkpoints/VGGnet_fast_rcnn_iter_50000.ckpt... done
 done.
Bus error: 10

I compile the code on MAC pro.

**Comment by eragonruan** [comment]
@Paleve can you provide more detail about the error

**Comment by Paleve** [comment]
@eragonruan 前面权重载入是没问题的，当前向计算的时候会报总线错误

2017-11-15 09:50:54.910148: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
2017-11-15 09:50:54.910152: W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
Tensor("Placeholder:0", shape=(?, ?, ?, 3), dtype=float32)
Tensor("conv5_3/conv5_3:0", shape=(?, ?, ?, 512), dtype=float32)
Tensor("rpn_conv/3x3/rpn_conv/3x3:0", shape=(?, ?, ?, 512), dtype=float32)
WARNING:tensorflow:<tensorflow.python.ops.rnn_cell_impl.BasicLSTMCell object at 0x10f73c290>: Using a concatenated state is slower and will soon be deprecated.  Use state_is_tuple=True.
Tensor("lstm_o/Reshape:0", shape=(?, ?, ?, 128), dtype=float32)
Tensor("lstm_o/Reshape:0", shape=(?, ?, ?, 128), dtype=float32)
Tensor("rpn_cls_score/Reshape:0", shape=(?, ?, ?, 20), dtype=float32)
Tensor("rpn_cls_prob:0", shape=(?, ?, ?, ?), dtype=float32)
Tensor("Reshape_5:0", shape=(?, ?, ?, 20), dtype=float32)
Tensor("rpn_bbox_pred/Reshape:0", shape=(?, ?, ?, 40), dtype=float32)
Tensor("Placeholder_1:0", shape=(?, 3), dtype=float32)
Loading network VGGnet_test...  Restoring from checkpoints/VGGnet_fast_rcnn_iter_50000.ckpt... done
 done.
Bus error: 10

**Comment by eragonruan** [comment]
@Paleve This may caused by your system, not the code. you can find more in these two issue: [tensorflow/issues/7663](https://github.com/tensorflow/tensorflow/issues/7663), [tensorflow/issues/6744](https://github.com/tensorflow/tensorflow/issues/6744)

**Comment by Paleve** [comment]
@eragonruan thanks a lot for your reply, however I meet a new issue , the loss is strange in the training. The original lose is around 2.0, but it increases to 50+ sometimes in the training, I think it's because the box isn't uniform splitted into 16  pixels' proposals, did you meet the similar problem before? I use ICDAR2017 MLT dataset. What's more, the box loss  decrease slowly.

**Comment by eragonruan** [comment]
@Paleve no, but sometimes the loss may jump to 8~10, this is caused by some "bad" images, because texts in mlt dataset contains oriented texts, if the rotated angle is too large, the loss may jump. box loss is hard to converge, in my experiments, you need to adjust the learning rate, and eventually box loss can reach to about 0.1 after 50k iterations.
